src/core/gltf2exporter/gltf2exporter.h

```diff
@@ -33,13 +33,50 @@
 #include <QUrl>
 #include <QDir>
 #include <QJsonObject>
+#include <QMap>
 #include <Kuesa/kuesa_global.h>
 
 QT_BEGIN_NAMESPACE
 
 namespace Kuesa {
 class GLTF2Context;
 class SceneEntity;
+
+class KUESASHARED_EXPORT GLTF2ExportConfiguration
+{
+public:
+    GLTF2ExportConfiguration();
+    ~GLTF2ExportConfiguration();
+    GLTF2ExportConfiguration(const GLTF2ExportConfiguration &other);
+    GLTF2ExportConfiguration(GLTF2ExportConfiguration &&other);
+    GLTF2ExportConfiguration &operator=(const GLTF2ExportConfiguration &other);
+    GLTF2ExportConfiguration &operator=(GLTF2ExportConfiguration &&other);
+
```

Comment:
Do you even need to define all of these if all are the default?

Reply:
actually no, I originally took inspiration from other Qt classes but they are not needed at all - removed.